src/Peekaqueue/SqsStatsService.cs

```diff
@@ -52,20 +61,24 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
             {
                 var queues = await GetQueuesToMonitor();
 
-                if (queues.Count == 0)
+                if (queues.Count > 0)
```

Comment:
You can reverse this logic to `queues.Count == 0`, exit early and avoid the nesting

Reply:
Done